frontends/web/src/components/headerssync/headerssync.module.css

```diff
@@ -1,7 +1,7 @@
 .syncContainer {
     position: relative;
     width: 100%;
-    top: -18px;
+    top: -4px;
     margin-bottom: -18px;
 }
```

Comment:
now there is a lot of space between the header and syncheader. I think a good alternative is to let the header become bigger. Can you change `height: var(--header-height);` to `min-height: var(--header-height);`, then this isn't needed.

  
![Screen Shot 2023-11-07 at 15 49 00](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/b165f879-56eb-4097-b997-c601ea18142d)